Index: Backend/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>DB_URL=jdbc:postgresql://localhost:5432/bank_application\r\nDB_USERNAME=postgres\r\nDB_PASSWORD=sqlpostgre
===================================================================
diff --git a/Backend/.env b/Backend/.env
--- a/Backend/.env	
+++ b/Backend/.env	
@@ -1,3 +1,3 @@
 DB_URL=jdbc:postgresql://localhost:5432/bank_application
 DB_USERNAME=postgres
-DB_PASSWORD=sqlpostgre
\ No newline at end of file
+DB_PASSWORD=postgres
\ No newline at end of file
